STEERING-COMMITTEE.md

```diff
@@ -0,0 +1,65 @@
+# Istio Steering Committee
+
+The Istio Steering Committee was formed to oversee the administrative aspects of the project, including governance, branding, and marketing. Steering was created to allow the Techincal Oversight Committee to exclusively focus on the technical aspects of the project. 
+
+## Steering Committee Charter
+
+1. Define, evolve, and defend the vision, values, mission, and scope of the project - to establish and maintain the soul of Istio
+2. Charter and refine policy for community groups (including Working Groups and Committees)
+3. Define and evolve project governance structures and policies, including how contributors become committers/maintainers, approvers, reviewers, members, code of conduct, etc.
+4. Control access to, establish processes regarding, and provide a final escalation path for any Istio repository
+5. Control and delegate access to and establish processes regarding other project resources/assets, including artifact repositories, build and test infrastructure, web sites and their domains, blogs, social-media accounts, etc.
+6. Manage the Istio brand to decide which things can be called “Istio” and how that mark can be used in relation to other efforts or vendors.
+7. Resolve any dispute from the Technical Oversight Committee.
+
+## Steering Committee Members
+
+Seats in Steering are held by a company, not by the individual users. Currently Steering consists of members from Google & IBM. These are:
+
+* [April Nassi](https://github.com/thisisnotapril) (Google)
```

Comment:
I wonder if we should organize these by company since the seats are company held?

We could do:

Google (6 seats):
* Googler 1
* Googler 2
...
IBM (3 Seats):
* IBMite 1
* IBMite 2
...

Would reinforce the "held by a company" aspect.

Reply:
Sounds good; will update